jans-linux-setup/jans_setup/setup_app/utils/base.py

```
@@ -381,20 +382,42 @@ def mylog(logs):
 
-    mylog("Downloading {} to {}".format(url, dst))
+    mylog(f"Downloading {url} to {dst}")
     download_tries = 1
+    download_ok = False
     while download_tries < 4:
         try:
             urllib.request.urlretrieve(url, dst)
             mylog("Download size: {} bytes".format(os.path.getsize(dst)))
             time.sleep(0.1)
+            download_ok = True
         except:
```

Ruff (0.15.21)

[error] Do not use bare `except`

(E722)